ocs_ci/ocs/resources/pvc.py

```diff
@@ -70,34 +70,38 @@ def resize_pvc(self, new_size, verify=False):
         return True
 
 
-def delete_all_pvcs():
+def delete_all_pvcs(namespace=None):
```

Comment:
please don't use this function like it is today.
This function should change to delete_pvcs and should get a specific list of pvc objects to be deleted

Reply:
changed accordingly